### PR TITLE
CI and quality check

### PR DESCRIPTION
**Pull request by bmarty**
Draft PR to check if CI is triggering properly

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -4,8 +4,78 @@ plugins {
...
+    detekt {
+        // preconfigure defaults
+        buildUponDefaultConfig = true
+        // activate all available (even unstable) rules.
+        allRules = true
+        // point to your custom config defining rules to run, overwriting default behavior
+        config = files("$rootDir/tools/detekt/detekt.yml")
+    }
+    dependencies {
+        detektPlugins("com.twitter.compose.rules:detekt:0.0.26")
```
Comment (ganfra) [inline build.gradle.kts:29]: Maybe slower to apply to all modules if a module doesn't contain compose?

**Review by ganfra — APPROVED** [review]
small remarks

File: build.gradle.kts [hunk 2]
```diff
@@ -4,8 +4,78 @@ plugins {
     alias(libs.plugins.android.library) apply false
     alias(libs.plugins.kotlin.android) apply false
     alias(libs.plugins.ksp) apply false
+    id("io.gitlab.arturbosch.detekt") version "1.22.0"
+    id("org.jlleitschuh.gradle.ktlint") version "11.0.0"
```
[the remark this answers is not in the document]
Comment (bmarty) [reply build.gradle.kts:8]: OK, moved to the catalog.

**Comment by bmarty** [comment]
Remaining issues are false positive, and the first one will be fixed by Dependabot hopefully.

**Pull request merged**